src/app/Main/Body/QueryPage/QueryPageContent/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -2,22 +2,73 @@
 import { css, jsx } from "@emotion/core";
 import { useTranslation } from "react-i18next";
 import { TFunction } from "i18next";
-import { ParameterContentListComponent } from "./ParameterContent-List.component";
+import { ListItem, ListItemText } from "@mui/material";
+import { ParameterListLayout } from "./ParameterList-Layout.component";
 import { AccordionComponent } from "../../../../../Shared/Accordion.component";
-import { FilterInterface } from "../../../../../Shared/Model/Filter.model";
+import {
+    FilterInterface,
+    FilterType,
+} from "../../../../../Shared/Model/Filter.model";
 import {
     DbKey,
     MainFilterList,
     subFiltersList,
 } from "../../../../../Shared/Model/Client_Isolate.model";
-import { createParameterListItemService } from "./createParameterListItem.service";
+import { getMicroorganismLabelService } from "../../Services/getMicroorganismLabel";
+import { replaceAll } from "../../../../../Core/replaceAll.service";
 
 const parameterBlockStyle = css`
     width: 100%;
     display: flex;
     flex-wrap: wrap;
     flex-grow: 1;
 `;
+const spaceStyle = css`
+    padding: 0;
+    margin: 0;
+`;
+const parameterValue = css`
+    margin-top: 0;
+    margin-left: 2em;
+    span {
+        letter-spacing: 0;
+    }
+`;
+
+function createParameterName(
+    parameterAttribute: FilterType,
+    parameter: string,
+    t: TFunction,
+    isSubFilter: boolean
+): { parameterName: string | JSX.Element; key: string } {
+    const key = parameter.replace(".", "");
+    let parameterName: string | JSX.Element = "";
+    if (parameterAttribute === "microorganism") {
+        const translateRootString = `FilterValues.formattedMicroorganisms.${key}`;
+        const prefix = t(`${translateRootString}.prefix`);
+        const name = t(`${translateRootString}.name`);
+        const italicName = t(`${translateRootString}.italicName`);
+        const suffix = t(`${translateRootString}.suffix`);
+        parameterName = getMicroorganismLabelService({
```

Comment:
So this is also a component-service mix, but that's another story...

Reply:
ok.